Watching for file changes with StatReloader
Error: That IP address can't be assigned to.
Traceback (most recent call last):
  File "/home/ec2-user/Django/Django_Project/manage.py", line 11, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ec2-user/Django/Django_Project/manage.py", line 22, in <module>
    main()
  File "/home/ec2-user/Django/Django_Project/manage.py", line 13, in main
    raise ImportError(
ImportError: Couldn't import Django. Are you sure it's installed and available on your PYTHONPATH environment variable? Did you forget to activate a virtual environment?
Traceback (most recent call last):
  File "/home/ec2-user/Django/Django_Project/manage.py", line 11, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ec2-user/Django/Django_Project/manage.py", line 22, in <module>
    main()
  File "/home/ec2-user/Django/Django_Project/manage.py", line 13, in main
    raise ImportError(
ImportError: Couldn't import Django. Are you sure it's installed and available on your PYTHONPATH environment variable? Did you forget to activate a virtual environment?
Traceback (most recent call last):
  File "/home/ec2-user/Django/Django_Project/manage.py", line 11, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ec2-user/Django/Django_Project/manage.py", line 22, in <module>
    main()
  File "/home/ec2-user/Django/Django_Project/manage.py", line 13, in main
    raise ImportError(
ImportError: Couldn't import Django. Are you sure it's installed and available on your PYTHONPATH environment variable? Did you forget to activate a virtual environment?
Unknown command: 'runs'. Did you mean runserver?
Type 'manage.py help' for usage.
Traceback (most recent call last):
  File "/home/ec2-user/Django/Django_Project/manage.py", line 11, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ec2-user/Django/Django_Project/manage.py", line 22, in <module>
    main()
  File "/home/ec2-user/Django/Django_Project/manage.py", line 13, in main
    raise ImportError(
ImportError: Couldn't import Django. Are you sure it's installed and available on your PYTHONPATH environment variable? Did you forget to activate a virtual environment?
Traceback (most recent call last):
  File "/home/ec2-user/Django/Django_Project/manage.py", line 11, in main
    from django.core.management import execute_from_command_line
ModuleNotFoundError: No module named 'django'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ec2-user/Django/Django_Project/manage.py", line 22, in <module>
    main()
  File "/home/ec2-user/Django/Django_Project/manage.py", line 13, in main
    raise ImportError(
ImportError: Couldn't import Django. Are you sure it's installed and available on your PYTHONPATH environment variable? Did you forget to activate a virtual environment?
Watching for file changes with StatReloader
Error: That port is already in use.
Watching for file changes with StatReloader
[08/Dec/2023 09:27:39] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 09:27:51] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 09:27:52] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 09:27:52] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 09:27:52] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 09:27:53] "GET /pybo/question/create/ HTTP/1.1" 200 2922
[08/Dec/2023 09:27:55] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 09:27:57] "GET /?kw=&page=1 HTTP/1.1" 200 5437
[08/Dec/2023 09:27:58] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 10:40:45] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 10:40:45] "GET /static/style.css HTTP/1.1" 304 0
[08/Dec/2023 10:40:45] "GET /static/bootstrap.min.css HTTP/1.1" 304 0
[08/Dec/2023 10:40:45] "GET /static/bootstrap.min.js HTTP/1.1" 304 0
[08/Dec/2023 10:40:46] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 10:40:55] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 10:40:56] "GET /pybo/ HTTP/1.1" 200 5437
[08/Dec/2023 10:40:58] "GET /pybo/question/create/ HTTP/1.1" 200 2922
Internal Server Error: /pybo/question/create/
Traceback (most recent call last):
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 477, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: attempt to write a readonly database

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/ec2-user/Django/Django_Project/pybo/views/question_views.py", line 18, in question_create
    question.save()
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 806, in save
    self.save_base(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 857, in save_base
    updated = self._save_table(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 1000, in _save_table
    results = self._do_insert(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 1041, in _do_insert
    return manager._insert(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/query.py", line 1434, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/sql/compiler.py", line 1621, in execute_sql
    cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 103, in execute
    return super().execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 477, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: attempt to write a readonly database
[08/Dec/2023 10:41:03] "POST /pybo/question/create/ HTTP/1.1" 500 145337
Internal Server Error: /pybo/question/create/
Traceback (most recent call last):
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 477, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: attempt to write a readonly database

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/ec2-user/Django/Django_Project/pybo/views/question_views.py", line 18, in question_create
    question.save()
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 806, in save
    self.save_base(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 857, in save_base
    updated = self._save_table(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 1000, in _save_table
    results = self._do_insert(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 1041, in _do_insert
    return manager._insert(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/query.py", line 1434, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/sql/compiler.py", line 1621, in execute_sql
    cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 103, in execute
    return super().execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 477, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: attempt to write a readonly database
[08/Dec/2023 10:41:08] "POST /pybo/question/create/ HTTP/1.1" 500 145337
Watching for file changes with StatReloader
[08/Dec/2023 10:43:34] "GET / HTTP/1.1" 200 5437
[08/Dec/2023 10:43:36] "GET /pybo/question/create/ HTTP/1.1" 200 2922
Internal Server Error: /pybo/question/create/
Traceback (most recent call last):
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 477, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: attempt to write a readonly database

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/home/ec2-user/Django/Django_Project/pybo/views/question_views.py", line 18, in question_create
    question.save()
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 806, in save
    self.save_base(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 857, in save_base
    updated = self._save_table(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 1000, in _save_table
    results = self._do_insert(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/base.py", line 1041, in _do_insert
    return manager._insert(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/query.py", line 1434, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/models/sql/compiler.py", line 1621, in execute_sql
    cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 103, in execute
    return super().execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "/home/ec2-user/Django/venv/lib64/python3.9/site-packages/django/db/backends/sqlite3/base.py", line 477, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: attempt to write a readonly database
[08/Dec/2023 10:43:38] "POST /pybo/question/create/ HTTP/1.1" 500 145283
Watching for file changes with StatReloader
Error: That port is already in use.
[08/Dec/2023 10:49:33] "GET /pybo/question/create/ HTTP/1.1" 302 0
[08/Dec/2023 10:49:33] "GET /common/login/?next=/pybo/question/create/ HTTP/1.1" 200 2871
[08/Dec/2023 10:49:44] "GET /common/signup/ HTTP/1.1" 200 3181
[08/Dec/2023 10:49:55] "POST /common/signup/ HTTP/1.1" 200 3588
[08/Dec/2023 10:50:02] "POST /common/signup/ HTTP/1.1" 302 0
[08/Dec/2023 10:50:02] "GET / HTTP/1.1" 200 5433
[08/Dec/2023 10:50:03] "GET /pybo/question/create/ HTTP/1.1" 200 2918
[08/Dec/2023 10:50:09] "POST /pybo/question/create/ HTTP/1.1" 302 0
[08/Dec/2023 10:50:09] "GET /pybo/ HTTP/1.1" 200 5978
[08/Dec/2023 10:50:35] "GET / HTTP/1.1" 200 6069
[08/Dec/2023 10:50:35] "GET /static/style.css HTTP/1.1" 200 79
[08/Dec/2023 10:50:35] "GET /static/bootstrap.min.js HTTP/1.1" 200 59219
[08/Dec/2023 10:50:35] "GET /static/bootstrap.min.css HTTP/1.1" 200 163873
Not Found: /favicon.ico
[08/Dec/2023 10:50:35] "GET /favicon.ico HTTP/1.1" 404 2438
[08/Dec/2023 10:50:37] "GET /pybo/question/create/ HTTP/1.1" 302 0
[08/Dec/2023 10:50:38] "GET /common/login/?next=/pybo/question/create/ HTTP/1.1" 200 2871
[08/Dec/2023 10:50:40] "GET /common/signup/ HTTP/1.1" 200 3181
[08/Dec/2023 10:51:27] "POST /common/signup/ HTTP/1.1" 302 0
[08/Dec/2023 10:51:27] "GET / HTTP/1.1" 200 5996
[08/Dec/2023 10:51:30] "GET /pybo/question/create/ HTTP/1.1" 200 2936
[08/Dec/2023 10:53:28] "POST /pybo/question/create/ HTTP/1.1" 302 0
[08/Dec/2023 10:53:28] "GET /pybo/ HTTP/1.1" 200 6720
[08/Dec/2023 10:53:32] "GET /pybo/2/ HTTP/1.1" 200 3553
[08/Dec/2023 10:53:35] "GET /pybo/2/ HTTP/1.1" 200 3553
[08/Dec/2023 10:53:44] "GET /pybo/question/vote/2/ HTTP/1.1" 302 0
[08/Dec/2023 10:53:44] "GET /pybo/2/ HTTP/1.1" 200 3816
[08/Dec/2023 10:53:56] "GET /static/bootstrap.min.css.map HTTP/1.1" 404 1824
[08/Dec/2023 10:54:01] "POST /pybo/answer/create/2/ HTTP/1.1" 302 0
[08/Dec/2023 10:54:01] "GET /pybo/2/ HTTP/1.1" 200 5131
[08/Dec/2023 10:54:01] "GET /static/bootstrap.min.css.map HTTP/1.1" 404 1824
[08/Dec/2023 10:54:41] "GET /static/bootstrap.min.css.map HTTP/1.1" 404 1824
[08/Dec/2023 10:54:51] "GET / HTTP/1.1" 200 6827
[08/Dec/2023 11:20:12] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x00{\x01\x00\x00w\x03\x03äÔdF6ðdÏkµô`g\\©Óá\x9eÃ\x87´BL¸kuN\x9f\x99åO\x91\x00\x00\x1aÀ/À+À\x11À\x07À\x13À')
[08/Dec/2023 11:20:12] You're accessing the development server over HTTPS, but it only supports HTTP.

[08/Dec/2023 11:20:13] code 400, message Bad request version ('À\x14À')
[08/Dec/2023 11:20:13] You're accessing the development server over HTTPS, but it only supports HTTP.

[08/Dec/2023 11:20:13] "GET / HTTP/1.1" 200 6900
[08/Dec/2023 11:20:14] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x00{\x01\x00\x00w\x03\x03\x10\x86Gþ\x89É\x95vÇ{Æa\x9fk*\x8dI\x83Ûz\x03«þ\x7f5pY\x9bêpq÷\x00\x00\x1aÀ/À+À\x11À\x07À\x13À')
[08/Dec/2023 11:20:14] You're accessing the development server over HTTPS, but it only supports HTTP.

[08/Dec/2023 11:28:14] "GET / HTTP/1.1" 200 6827
[08/Dec/2023 11:38:05] code 400, message Bad request syntax ('JDWP-Handshake')
[08/Dec/2023 11:38:05] "JDWP-Handshake" 400 -
[08/Dec/2023 11:59:01] "GET / HTTP/1.1" 200 6900
Not Found: /favicon.ico
[08/Dec/2023 11:59:41] "GET /favicon.ico HTTP/1.1" 404 2438
Not Found: /http:/api.ipify.org/
[08/Dec/2023 12:00:05] "GET http://api.ipify.org/?format=json HTTP/1.1" 404 1818
Not Found: /www.shadowserver.org:443
[08/Dec/2023 12:00:27] "CONNECT www.shadowserver.org:443 HTTP/1.1" 404 1827
[08/Dec/2023 12:50:33] "GET / HTTP/1.1" 200 6827
[08/Dec/2023 12:50:33] "GET /static/style.css HTTP/1.1" 304 0
[08/Dec/2023 12:50:33] "GET /static/bootstrap.min.css HTTP/1.1" 304 0
[08/Dec/2023 12:50:33] "GET /static/bootstrap.min.js HTTP/1.1" 304 0
[08/Dec/2023 12:58:09] code 400, message Bad request version ('À\x13À')
[08/Dec/2023 12:58:09] You're accessing the development server over HTTPS, but it only supports HTTP.

[08/Dec/2023 12:58:09] code 400, message Bad request version ('À(À$À\x14À')
[08/Dec/2023 12:58:09] You're accessing the development server over HTTPS, but it only supports HTTP.

Not Found: /http:/3.37.164.41:8000/phpmyadmin/scripts/setup.php
[08/Dec/2023 14:33:30] "GET http://3.37.164.41:8000/phpmyadmin/scripts/setup.php HTTP/1.0" 404 1939
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.11.4/scripts/setup.php
[08/Dec/2023 14:33:30] "GET http://3.37.164.41:8000/phpMyAdmin-2.11.4/scripts/setup.php HTTP/1.0" 404 1960
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.11.3/scripts/setup.php
[08/Dec/2023 14:33:31] "GET http://3.37.164.41:8000/phpMyAdmin-2.11.3/scripts/setup.php HTTP/1.0" 404 1960
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.10.0.2/scripts/setup.php
[08/Dec/2023 14:33:31] "GET http://3.37.164.41:8000/phpMyAdmin-2.10.0.2/scripts/setup.php HTTP/1.0" 404 1966
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.10.3/scripts/setup.php
[08/Dec/2023 14:33:31] "GET http://3.37.164.41:8000/phpMyAdmin-2.10.3/scripts/setup.php HTTP/1.0" 404 1960
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.8.0.2/scripts/setup.php
[08/Dec/2023 14:33:32] "GET http://3.37.164.41:8000/phpMyAdmin-2.8.0.2/scripts/setup.php HTTP/1.0" 404 1963
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.10.2/scripts/setup.php
[08/Dec/2023 14:33:32] "GET http://3.37.164.41:8000/phpMyAdmin-2.10.2/scripts/setup.php HTTP/1.0" 404 1960
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.11.0/scripts/setup.php
[08/Dec/2023 14:35:43] "GET http://3.37.164.41:8000/phpMyAdmin-2.11.0/scripts/setup.php HTTP/1.0" 404 1960
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.11.7/scripts/setup.php
[08/Dec/2023 14:35:43] "GET http://3.37.164.41:8000/phpMyAdmin-2.11.7/scripts/setup.php HTTP/1.0" 404 1960
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.11.1.2/scripts/setup.php
[08/Dec/2023 14:35:44] "GET http://3.37.164.41:8000/phpMyAdmin-2.11.1.2/scripts/setup.php HTTP/1.0" 404 1966
Not Found: /http:/3.37.164.41:8000/pma/scripts/setup.php
[08/Dec/2023 14:35:44] "GET http://3.37.164.41:8000/pma/scripts/setup.php HTTP/1.0" 404 1918
Not Found: /http:/3.37.164.41:8000/phpMyAdmin3/scripts/setup.php
[08/Dec/2023 14:35:44] "GET http://3.37.164.41:8000/phpMyAdmin3/scripts/setup.php HTTP/1.0" 404 1942
Not Found: /http:/3.37.164.41:8000/myadmin/scripts/setup.php
[08/Dec/2023 14:35:45] "GET http://3.37.164.41:8000/myadmin/scripts/setup.php HTTP/1.0" 404 1930
Not Found: /http:/3.37.164.41:8000/MyAdmin/scripts/setup.php
[08/Dec/2023 14:35:48] "GET http://3.37.164.41:8000/MyAdmin/scripts/setup.php HTTP/1.0" 404 1930
Not Found: /http:/3.37.164.41:8000/PHPMYADMIN/scripts/setup.php
[08/Dec/2023 14:35:49] "GET http://3.37.164.41:8000/PHPMYADMIN/scripts/setup.php HTTP/1.0" 404 1939
Not Found: /http:/3.37.164.41:8000/mysqladmin/scripts/setup.php
[08/Dec/2023 14:35:49] "GET http://3.37.164.41:8000/mysqladmin/scripts/setup.php HTTP/1.0" 404 1939
Not Found: /http:/3.37.164.41:8000/SQL/scripts/setup.php
[08/Dec/2023 14:35:49] "GET http://3.37.164.41:8000/SQL/scripts/setup.php HTTP/1.0" 404 1918
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.5.5-pl1/scripts/setup.php
[08/Dec/2023 14:35:50] "GET http://3.37.164.41:8000/phpMyAdmin-2.5.5-pl1/scripts/setup.php HTTP/1.0" 404 1969
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.5.5/scripts/setup.php
[08/Dec/2023 14:35:50] "GET http://3.37.164.41:8000/phpMyAdmin-2.5.5/scripts/setup.php HTTP/1.0" 404 1957
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.5.4/scripts/setup.php
[08/Dec/2023 14:35:50] "GET http://3.37.164.41:8000/phpMyAdmin-2.5.4/scripts/setup.php HTTP/1.0" 404 1957
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2.5.7-pl1/scripts/setup.php
[08/Dec/2023 14:35:51] "GET http://3.37.164.41:8000/phpMyAdmin-2.5.7-pl1/scripts/setup.php HTTP/1.0" 404 1969
Not Found: /http:/3.37.164.41:8000/admin/pma/scripts/setup.php
[08/Dec/2023 14:35:51] "GET http://3.37.164.41:8000/admin/pma/scripts/setup.php HTTP/1.0" 404 1936
Not Found: /http:/3.37.164.41:8000/phpMyAdmin-2/scripts/setup.php
[08/Dec/2023 14:35:51] "GET http://3.37.164.41:8000/phpMyAdmin-2/scripts/setup.php HTTP/1.0" 404 1945
Not Found: /http:/3.37.164.41:8000/web/phpMyAdmin/scripts/setup.php
[08/Dec/2023 14:35:52] "GET http://3.37.164.41:8000/web/phpMyAdmin/scripts/setup.php HTTP/1.0" 404 1951
Not Found: /http:/3.37.164.41:8000/webadmin/scripts/setup.php
[08/Dec/2023 14:35:53] "GET http://3.37.164.41:8000/webadmin/scripts/setup.php HTTP/1.0" 404 1933
Not Found: /http:/3.37.164.41:8000/admin/scripts/setup.php
[08/Dec/2023 14:35:53] "GET http://3.37.164.41:8000/admin/scripts/setup.php HTTP/1.0" 404 1924
Not Found: /http:/3.37.164.41:8000/dbadmin/scripts/setup.php
[08/Dec/2023 14:35:54] "GET http://3.37.164.41:8000/dbadmin/scripts/setup.php HTTP/1.0" 404 1930
Not Found: /http:/3.37.164.41:8000/mysql/scripts/setup.php
[08/Dec/2023 14:35:54] "GET http://3.37.164.41:8000/mysql/scripts/setup.php HTTP/1.0" 404 1924
Not Found: /http:/3.37.164.41:8000/phpMyAdmin2/scripts/setup.php
[08/Dec/2023 14:35:54] "GET http://3.37.164.41:8000/phpMyAdmin2/scripts/setup.php HTTP/1.0" 404 1942
Not Found: /http:/3.37.164.41:8000/phpma/scripts/setup.php
[08/Dec/2023 14:35:55] "GET http://3.37.164.41:8000/phpma/scripts/setup.php HTTP/1.0" 404 1924
Not Found: /http:/3.37.164.41:8000/sqlweb/scripts/setup.php
[08/Dec/2023 14:35:55] "GET http://3.37.164.41:8000/sqlweb/scripts/setup.php HTTP/1.0" 404 1927
Not Found: /http:/3.37.164.41:8000/webdb/scripts/setup.php
[08/Dec/2023 14:35:57] "GET http://3.37.164.41:8000/webdb/scripts/setup.php HTTP/1.0" 404 1924
Not Found: /http:/3.37.164.41:8000/websql/scripts/setup.php
[08/Dec/2023 14:35:58] "GET http://3.37.164.41:8000/websql/scripts/setup.php HTTP/1.0" 404 1927
Not Found: /http:/3.37.164.41:8000/_phpMyAdmin/scripts/setup.php
[08/Dec/2023 14:36:00] "GET http://3.37.164.41:8000/_phpMyAdmin/scripts/setup.php HTTP/1.0" 404 1942
Not Found: /http:/3.37.164.41:8000/php/scripts/setup.php
[08/Dec/2023 14:36:00] "GET http://3.37.164.41:8000/php/scripts/setup.php HTTP/1.0" 404 1918
Not Found: /http:/3.37.164.41:8000/admin/phpmyadmin/scripts/setup.txt
[08/Dec/2023 14:36:09] "GET http://3.37.164.41:8000/admin/phpmyadmin/scripts/setup.txt HTTP/1.0" 404 1957
Not Found: /http:/3.37.164.41:8000/db/scripts/setup.php
[08/Dec/2023 14:36:09] "GET http://3.37.164.41:8000/db/scripts/setup.php HTTP/1.0" 404 1915
Not Found: /http:/3.37.164.41:8000/sqlmanager/scripts/setup.php
[08/Dec/2023 14:36:10] "GET http://3.37.164.41:8000/sqlmanager/scripts/setup.php HTTP/1.0" 404 1939
Not Found: /http:/3.37.164.41:8000/mysqlmanager/scripts/setup.php
[08/Dec/2023 14:36:20] "GET http://3.37.164.41:8000/mysqlmanager/scripts/setup.php HTTP/1.0" 404 1945
Not Found: /http:/3.37.164.41:8000/phpmanager/scripts/setup.php
[08/Dec/2023 14:36:21] "GET http://3.37.164.41:8000/phpmanager/scripts/setup.php HTTP/1.0" 404 1939
Not Found: /http:/3.37.164.41:8000/php-myadmin/scripts/setup.php
[08/Dec/2023 14:36:24] "GET http://3.37.164.41:8000/php-myadmin/scripts/setup.php HTTP/1.0" 404 1942
Not Found: /http:/3.37.164.41:8000/phpmy-admin/scripts/setup.php
[08/Dec/2023 14:36:24] "GET http://3.37.164.41:8000/phpmy-admin/scripts/setup.php HTTP/1.0" 404 1942
Not Found: /http:/3.37.164.41:8000/mysql-admin/scripts/setup.php
[08/Dec/2023 14:36:25] "GET http://3.37.164.41:8000/mysql-admin/scripts/setup.php HTTP/1.0" 404 1942
[08/Dec/2023 15:21:58] "GET / HTTP/1.1" 200 6809
[08/Dec/2023 15:21:58] "GET /static/bootstrap.min.css HTTP/1.1" 304 0
[08/Dec/2023 15:21:58] "GET /static/style.css HTTP/1.1" 304 0
[08/Dec/2023 15:21:58] "GET /static/bootstrap.min.js HTTP/1.1" 304 0
[08/Dec/2023 15:22:06] "GET /pybo/2/ HTTP/1.1" 200 4567
[08/Dec/2023 15:22:11] "GET /pybo/answer/vote/1/ HTTP/1.1" 302 0
[08/Dec/2023 15:22:11] "GET /pybo/2/ HTTP/1.1" 200 4567
[08/Dec/2023 15:22:13] "GET /pybo/question/vote/2/ HTTP/1.1" 302 0
[08/Dec/2023 15:22:14] "GET /pybo/2/ HTTP/1.1" 200 4567
[08/Dec/2023 15:45:19] "GET / HTTP/1.1" 200 6900
[08/Dec/2023 16:08:15] code 400, message Bad request version ('\x02h2\x08http/1.1\x00\x1b\x00\x03\x02\x00\x02\x00-\x00\x02\x01\x01\x00')
[08/Dec/2023 16:08:15] You're accessing the development server over HTTPS, but it only supports HTTP.

[08/Dec/2023 16:08:15] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x02:\x01\x00\x026\x03\x03¨L\x99\x8aäà¾*G\x91\x8f\x93O\xadgf!\x87X')
[08/Dec/2023 16:08:15] You're accessing the development server over HTTPS, but it only supports HTTP.

[08/Dec/2023 16:08:15] "GET / HTTP/1.1" 200 6900
[08/Dec/2023 16:08:15] "GET /static/bootstrap.min.css HTTP/1.1" 200 163873
[08/Dec/2023 16:08:15] "GET /static/style.css HTTP/1.1" 200 79
[08/Dec/2023 16:08:15] "GET /static/bootstrap.min.js HTTP/1.1" 200 59219
Not Found: /favicon.ico
[08/Dec/2023 16:08:15] "GET /favicon.ico HTTP/1.1" 404 2438
[08/Dec/2023 17:48:13] "GET / HTTP/1.1" 200 6900
[08/Dec/2023 21:07:46] "GET / HTTP/1.1" 200 6900
[08/Dec/2023 21:41:12,393] - Broken pipe from ('167.94.138.127', 59356)

[08/Dec/2023 21:41:15] "GET / HTTP/1.1" 200 6900
[08/Dec/2023 21:41:15,782] - Broken pipe from ('167.94.138.127', 49078)

[08/Dec/2023 21:41:18] "GET / HTTP/1.1" 200 6900
[08/Dec/2023 21:41:19] code 505, message Invalid HTTP version (2.0)
[08/Dec/2023 21:41:19] "PRI * HTTP/2.0" 505 -
Not Found: /favicon.ico
[08/Dec/2023 21:41:19] "GET /favicon.ico HTTP/1.1" 404 2438
[08/Dec/2023 22:25:22] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 00:31:24] "GET / HTTP/1.1" 200 6900
Not Found: /favicon.ico
[09/Dec/2023 00:31:40] "GET /favicon.ico HTTP/1.1" 404 2438
Not Found: /http:/api.ipify.org/
[09/Dec/2023 00:31:45] "GET http://api.ipify.org/?format=json HTTP/1.1" 404 1818
Not Found: /www.shadowserver.org:443
[09/Dec/2023 00:31:51] "CONNECT www.shadowserver.org:443 HTTP/1.1" 404 1827
[09/Dec/2023 03:27:52,684] - Broken pipe from ('167.94.138.124', 56388)

[09/Dec/2023 03:27:55] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 03:27:56,069] - Broken pipe from ('167.94.138.124', 34008)

[09/Dec/2023 03:27:59] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 03:27:59] code 505, message Invalid HTTP version (2.0)
[09/Dec/2023 03:27:59] "PRI * HTTP/2.0" 505 -
Not Found: /favicon.ico
[09/Dec/2023 03:28:00] "GET /favicon.ico HTTP/1.1" 404 2438
[09/Dec/2023 03:37:27] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 03:37:29] "GET /static/style.css HTTP/1.1" 200 79
[09/Dec/2023 03:37:29] "GET /static/bootstrap.min.css HTTP/1.1" 200 163873
[09/Dec/2023 03:37:29] "GET /static/bootstrap.min.js HTTP/1.1" 200 59219
Not Found: /favicon.ico
[09/Dec/2023 03:37:29] "GET /favicon.ico HTTP/1.1" 404 2438
[09/Dec/2023 03:56:35] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x00{\x01\x00\x00w\x03\x03ú\x9aMø\x7f:¬FÙs2a\x8eú\x98æÚm\x06\x14vó°¿+$Ó\x8aHº²\x89\x00\x00\x1aÀ/À+À\x11À\x07À\x13À')
[09/Dec/2023 03:56:35] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 03:56:35] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x00{\x01\x00\x00w\x03\x03çþ@\x19nO¾Z¶ÔhLï³XM!=EJëmO\x88\x11_¾÷/Kª]\x00\x00\x1aÀ/À+À\x11À\x07À\x13À')
[09/Dec/2023 03:56:35] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 03:56:36] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 03:56:37] code 400, message Bad request version ('À\x14À')
[09/Dec/2023 03:56:37] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 04:38:01] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 04:38:01,433] - Broken pipe from ('152.32.247.130', 34590)

[09/Dec/2023 04:38:01] code 400, message Bad request version ('À\x13À')
[09/Dec/2023 04:38:01] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 04:38:01] code 400, message Bad HTTP/0.9 request type ('t3')
[09/Dec/2023 04:38:01] "t3 12.1.2" 400 -
[09/Dec/2023 04:38:10] "GET / HTTP/1.1" 200 6900
Not Found: /favicon.ico
[09/Dec/2023 04:38:11] "GET /favicon.ico HTTP/1.1" 404 2438
Not Found: /sitemap.xml
Not Found: /robots.txt
[09/Dec/2023 04:38:11] "GET /sitemap.xml HTTP/1.1" 404 2438
[09/Dec/2023 04:38:11] "GET /robots.txt HTTP/1.1" 404 2435
[09/Dec/2023 07:39:22] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 07:40:47] code 400, message Bad request version ('À\x13À')
[09/Dec/2023 07:40:47] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 07:40:47] code 400, message Bad request version ('À(À$À\x14À')
[09/Dec/2023 07:40:47] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 09:19:14] code 400, message Bad request syntax ('nil')
[09/Dec/2023 09:19:14] "nil" 400 -
[09/Dec/2023 09:19:15] "GET / HTTP/1.0" 200 6900
[09/Dec/2023 09:19:15] "GET / HTTP/1.1" 200 6900
Not Found: /sitemap.xml
[09/Dec/2023 09:19:15] "GET /sitemap.xml HTTP/1.1" 404 2438
Not Found: /robots.txt
[09/Dec/2023 09:19:15] "GET /robots.txt HTTP/1.1" 404 2435
[09/Dec/2023 10:36:29] "GET /static/bootstrap.min.css HTTP/1.1" 304 0
[09/Dec/2023 10:51:04] code 400, message Bad request version ('sóqJª\x17¯|?Z\x13+Y\x8e(ó·Z\x00°%')
[09/Dec/2023 10:51:04] code 400, message Bad request version ('\x7fØE\x00°\x96\x02þdî\x1aiÝ¥R-ÿ7¼H¶6\x80\x9b\x05F\x19\x91K')
[09/Dec/2023 10:51:04] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:04] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:04] code 400, message Bad request version ('EÝ\x8eá\x9aq¤.uA\x00\x1b\x00\x03\x02\x00\x02\x00')
[09/Dec/2023 10:51:04] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:04] code 400, message Bad request version ('\x00\x02\x01\x00\x00#\x00\x00\x00\x17\x00\x00\x00\x05\x00\x05\x01\x00\x00\x00\x00\x00')
[09/Dec/2023 10:51:04] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:04] code 400, message Bad request version ('\x02h2\x08http/1.1\x00#\x00\x00\x00')
[09/Dec/2023 10:51:04] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:04] code 400, message Bad request version ('\x00\x02\x01\x00\x00')
[09/Dec/2023 10:51:04] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] code 400, message Bad request version ('AÍ\x03|¦÷\x999"S¹oyP')
[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] code 400, message Bad request version ('?)¡d2n')
[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] code 400, message Bad request version ('-\x14i\x14«Þ;f\x0ecj\x0fßØ\x99%«e\x9d8IÊ©ìÞì\x82õÔ:W!\x18\x19\x03Éùa\x9bùÜ\x16¨ê_')
[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] code 400, message Bad request version ('\x02h2\x08http/1.1\x00\x05\x00\x05\x01\x00\x00\x00\x00\x00\x12\x00\x00\x00')
[09/Dec/2023 10:51:06] code 400, message Bad request version ('!Þ\x13â6ïR~>\x12¥j\x12;%')
[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] code 400, message Bad request version ('\x00\x12\x00\x10\x04\x03\x08\x04\x04\x01\x05\x03\x08\x05\x05\x01\x08\x06\x06\x01\x00+\x00\x07\x06ZZ\x03\x04\x03\x03\x00')
[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] code 400, message Bad request version ('ç0Etc\x08V^\x1a')
[09/Dec/2023 10:51:06] code 400, message Bad request version ('\x02h2\x08http/1.1\x00')
[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] code 400, message Bad request version ('õ')
[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] code 400, message Bad request version ('\x9a\x9a\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x01\x93ÊÊ\x00\x00\x00')
[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] code 400, message Bad request version ('\x00\x12\x00\x10\x04\x03\x08\x04\x04\x01\x05\x03\x08\x05\x05\x01\x08\x06\x06\x01\x00')
[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:06] code 400, message Bad request version ('&Ý¶ss\x0eP>"\x83ðná5\x00')
[09/Dec/2023 10:51:06] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('Æ`»#\x87ÌdËvvZ\xadi¨oJ\x81\x94´õþ~à¢7DA\x86êìW-\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x02h2\x08http/1.1\x00-\x00\x02\x01\x01ÿ\x01\x00\x01\x00\x00')
[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x8b|p')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x00\x12\x00\x10\x04\x03\x08\x04\x04\x01\x05\x03\x08\x05\x05\x01\x08\x06\x06\x01\x00\x05\x00\x05\x01\x00\x00\x00\x00\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x8f&\x87*Âü\x83\x9fdÖ\x8b\x93ÔúNJÏndt\x02CA·ß\x8a°rèYh\xadUè\x12d0#Or÷\x8et\x05A¦')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('¯¯hÄú©\x82\x94Ë\x06¯;v\x13¹4\x01¯ÝÂ\x118þs"â:¦\x15½V&\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x00\x02\x01\x00Di\x00\x05\x00\x03\x02h2\x00+\x00\x07\x06::\x03\x04\x03\x03\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x00#\x00\x00\x00-\x00\x02\x01\x01\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x8a\x8a\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x01Íºº\x00\x00Di\x00\x05\x00\x03\x02h2\x003\x00+\x00)')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x02h2\x08http/1.1\x00+\x00\x07\x06')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('»\x16[à5Þsã')
[09/Dec/2023 10:51:07] code 400, message Bad request syntax ('\x16\x03\x01\x02\x00\x01\x00\x01ü\x03\x03RÇA\x9d')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('Sùæjqe1\x83nÓ4\x815\x08\x9f@øùÃ!\x04\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x8a\x8a\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x01\xadÚÚ\x00\x00\x00-\x00\x02\x01\x01\x00\x17\x00\x00\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('+')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('íï\x07\x1b©1')
[09/Dec/2023 10:51:07] code 400, message Bad request version ('ªª\x13\x01\x13\x02\x13\x03À+À/À,À0Ì©Ì¨À\x13À\x14\x00\x9c\x00\x9d\x00/\x005\x01\x00\x01\x93\x8a\x8a\x00\x00\x00#\x00\x00ÿ\x01\x00\x01\x00\x00+\x00\x07\x06ªª\x03\x04\x03\x03\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x00éÌX\x00\x12\x00\x00\x00-\x00\x02\x01\x01Di\x00\x05\x00\x03\x02h2\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('TõyKòKv\x04Q\x86Í\x90~º\x99S')
[09/Dec/2023 10:51:07] code 400, message Bad request version ('Ðdý¬k0L\x00\x1b\x00\x03\x02\x00\x02\x00\x05\x00\x05\x01\x00\x00\x00\x00\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('ð¶+Ï©å\x82Ëð\x13íM¯U4qY02I\x81B\x97\\@EÍ}¼|\x92L\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('÷km\x96ú±¶\x00n¥6~ÀÇ\x9bS')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] code 400, message Bad request version ('öa¥#äë6\x90ÊÚ')
[09/Dec/2023 10:51:07] code 400, message Bad request version ('\x00\x12\x00\x10\x04\x03\x08\x04\x04\x01\x05\x03\x08\x05\x05\x01\x08\x06\x06\x01\x00#\x00\x00\x00')
[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:07] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:12] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x02\x00\x01\x00\x01ü\x03\x03\x14\x82')
[09/Dec/2023 10:51:12] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:12] code 400, message Bad request syntax ('\x16\x03\x01\x02:\x01\x00\x026\x03\x03@pJqJì3]ß9i¶Òå÷Gz\x01ø\x9er¬t')
[09/Dec/2023 10:51:12] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:12] code 400, message Bad request version ('0²ýE\x81ÁqÀåiJ9\x03»ä+;¶è¼')
[09/Dec/2023 10:51:12] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:12] code 400, message Bad request version ('\x00')
[09/Dec/2023 10:51:12] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:12] code 400, message Bad request version ('="ô\\\x07\x80ÈµòòV»h\x97+K\x92Øh\x1bËÔòUÛKKJïn\x9d\x11\x00\x17\x00\x00\x00')
[09/Dec/2023 10:51:12] code 400, message Bad request version ('Ö\x819\x1a')
[09/Dec/2023 10:51:12] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:12] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:16] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x03\x00«\x01\x00\x00§\x03\x03etF¡tÕhïù)\x90{')
[09/Dec/2023 10:51:16] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:16,093] - Broken pipe from ('125.141.87.209', 3868)

[09/Dec/2023 10:51:24] "GET /static/style.css HTTP/1.1" 304 0
[09/Dec/2023 10:51:24] "GET /static/bootstrap.min.js HTTP/1.1" 304 0
[09/Dec/2023 10:51:25] "GET / HTTP/1.1" 200 6809
[09/Dec/2023 10:51:26] "GET / HTTP/1.1" 200 6809
[09/Dec/2023 10:51:26] "GET / HTTP/1.1" 200 6809
[09/Dec/2023 10:51:26] "GET / HTTP/1.1" 200 6809
[09/Dec/2023 10:51:30] code 400, message Bad request version ('|\x94\x19\x94\x7fÿ\x9a%Íb£¸9y¦')
[09/Dec/2023 10:51:30] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:30] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 10:51:30] code 400, message Bad request version ('\x00\x02\x01\x00\x00\x1b\x00\x03\x02\x00\x02\x00-\x00\x02\x01\x01Di\x00\x05\x00\x03\x02h2\x00#\x00\x00\x003\x00+\x00)')
[09/Dec/2023 10:51:30] You're accessing the development server over HTTPS, but it only supports HTTP.

[09/Dec/2023 10:51:30] "GET /static/style.css HTTP/1.1" 200 79
[09/Dec/2023 10:51:30] "GET /static/bootstrap.min.css HTTP/1.1" 200 163873
[09/Dec/2023 10:51:30] "GET /static/bootstrap.min.js HTTP/1.1" 200 59219
Not Found: /favicon.ico
[09/Dec/2023 10:51:30] "GET /favicon.ico HTTP/1.1" 404 2438
[09/Dec/2023 10:52:00,179] - Broken pipe from ('199.45.154.48', 59618)

[09/Dec/2023 10:52:03] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 10:52:03,695] - Broken pipe from ('199.45.154.48', 59630)

[09/Dec/2023 10:52:07] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 10:52:08] code 505, message Invalid HTTP version (2.0)
[09/Dec/2023 10:52:08] "PRI * HTTP/2.0" 505 -
Not Found: /favicon.ico
[09/Dec/2023 10:52:08] "GET /favicon.ico HTTP/1.1" 404 2438
[09/Dec/2023 11:07:34] "GET /common/login/ HTTP/1.1" 200 2849
[09/Dec/2023 11:07:38] "POST /common/login/ HTTP/1.1" 302 0
[09/Dec/2023 11:07:38] "GET / HTTP/1.1" 200 6813
[09/Dec/2023 11:07:45] "GET /admin HTTP/1.1" 301 0
[09/Dec/2023 11:07:45] "GET /admin/ HTTP/1.1" 200 4387
[09/Dec/2023 11:07:45] "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2616
[09/Dec/2023 11:07:45] "GET /static/admin/css/base.css HTTP/1.1" 200 19513
[09/Dec/2023 11:07:45] "GET /static/admin/css/dashboard.css HTTP/1.1" 200 380
[09/Dec/2023 11:07:45] "GET /static/admin/css/responsive.css HTTP/1.1" 200 18575
[09/Dec/2023 11:07:45] "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3401
[09/Dec/2023 11:07:45] "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
[09/Dec/2023 11:07:45] "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
[09/Dec/2023 11:07:45] "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
[09/Dec/2023 11:07:45] "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 86184
[09/Dec/2023 11:07:45] "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 85692
[09/Dec/2023 11:07:45] "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 85876
[09/Dec/2023 11:07:51] "GET /admin/pybo/question/ HTTP/1.1" 200 7100
[09/Dec/2023 11:07:51] "GET /static/admin/css/changelists.css HTTP/1.1" 200 6932
[09/Dec/2023 11:07:51] "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
[09/Dec/2023 11:07:51] "GET /static/admin/js/core.js HTTP/1.1" 200 5698
[09/Dec/2023 11:07:51] "GET /static/admin/js/actions.js HTTP/1.1" 200 7872
[09/Dec/2023 11:07:51] "GET /admin/jsi18n/ HTTP/1.1" 200 3343
[09/Dec/2023 11:07:51] "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 5984
[09/Dec/2023 11:07:51] "GET /static/admin/js/urlify.js HTTP/1.1" 200 7902
[09/Dec/2023 11:07:51] "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
[09/Dec/2023 11:07:51] "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 288580
[09/Dec/2023 11:07:51] "GET /static/admin/img/search.svg HTTP/1.1" 200 458
[09/Dec/2023 11:07:51] "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 232381
[09/Dec/2023 11:07:51] "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
[09/Dec/2023 11:07:57] "GET /admin/pybo/question/add/ HTTP/1.1" 200 10172
[09/Dec/2023 11:07:57] "GET /static/admin/css/forms.css HTTP/1.1" 200 8878
[09/Dec/2023 11:07:57] "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 19634
[09/Dec/2023 11:07:57] "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 492
[09/Dec/2023 11:07:57] "GET /admin/jsi18n/ HTTP/1.1" 200 3343
[09/Dec/2023 11:07:57] "GET /static/admin/js/calendar.js HTTP/1.1" 200 8466
[09/Dec/2023 11:07:57] "GET /static/admin/css/widgets.css HTTP/1.1" 200 11097
[09/Dec/2023 11:07:57] "GET /static/admin/js/change_form.js HTTP/1.1" 200 606
[09/Dec/2023 11:07:57] "GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677
[09/Dec/2023 11:07:57] "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086
[09/Dec/2023 11:08:04] "GET /admin/auth/user/add/?_to_field=id&_popup=1 HTTP/1.1" 200 4612
[09/Dec/2023 11:08:04] "GET /admin/jsi18n/ HTTP/1.1" 200 3343
[09/Dec/2023 11:08:18] "GET /admin/auth/user/add/?_to_field=id&_popup=1 HTTP/1.1" 200 4612
[09/Dec/2023 11:08:18] "GET /admin/jsi18n/ HTTP/1.1" 200 3343
[09/Dec/2023 11:14:52] "GET / HTTP/1.1" 200 6813
[09/Dec/2023 11:14:54] "GET /pybo/2/ HTTP/1.1" 200 4567
[09/Dec/2023 11:14:56] "GET /pybo/30 HTTP/1.1" 301 0
Not Found: /pybo/30/
[09/Dec/2023 11:14:56] "GET /pybo/30/ HTTP/1.1" 404 2628
[09/Dec/2023 11:30:01] "GET / HTTP/1.1" 200 6900
[09/Dec/2023 12:25:12] "GET /pybo/ HTTP/1.1" 200 6813
[09/Dec/2023 13:24:27] code 400, message Bad request syntax ('MGLNDD_3.37.164.41_8000')
[09/Dec/2023 13:24:27] "MGLNDD_3.37.164.41_8000" 400 -
